lib/plugins/filter/after_post_render/external_link.js

```diff
@@ -2,7 +2,7 @@
 
 const { isExternalLink } = require('hexo-util');
 let EXTERNAL_LINK_POST_ENABLED = true;
-const rATag = /<a(?:\s+?|\s+?[^<>]+\s+?)?href=["']((?:\/\/|http(?:s)?:)[^<>"']+)["'][^<>]*>/gi;
+const rATag = /<a(?:\s+?|\s+?[^<>]+\s+?)?href=["']((?:http(?:s)?:|\/\/)[^<>"']+)["'][^<>]*>/gi;
```

Comment:
It is more common to have `https://` and `http://` inside links instead of `//`. By moving `\/\/` backward it is a little bit faster.